src/osal/lv_freertos.h

```diff
@@ -63,6 +63,7 @@ typedef struct {
 typedef struct {
 #if USE_FREERTOS_TASK_NOTIFY
     TaskHandle_t xTaskToNotify;
+    BaseType_t xSignalSent;
```

Comment:
Please use the freertos naming convention all over in this file:
https://www.freertos.org/Documentation/02-Kernel/06-Coding-guidelines/02-FreeRTOS-Coding-Standard-and-Style-Guide#naming-conventions

At a better look, I think you can use a member like 
`BaseType_t    xIsInitialized; `
similar as it is done on semaphore implementation.

E.G. no signal can be done until the initialization is done.